src/solver/ts-generator/include/antares/solver/ts-generator/prepro.h

```diff
@@ -24,14 +24,15 @@
 #include <antares/array/matrix.h>
 #include <antares/study/fwd.h>
 #include <antares/study/parts/thermal/defines.h>
+#include <antares/solver/ts-generator/law.h>
 #include <memory>
 
 namespace Antares::Data
 {
 /*!
 ** \brief Thermal
 */
-class PreproThermal
+class PreproAvailability
```

Comment:
**PreproAvailability** : watch for max value, which is **thermalPreproMax**

Reply:
[done](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2019/commits/62a87063ff4648c0b45ba6c3830a4ac0509a0715)